demisto_sdk/commands/common/hook_validations/incident_field.py

```diff
@@ -356,3 +358,15 @@ def is_changed_type(self):
                     is_type_changed = True
 
         return is_type_changed
+
+    def is_valid_grid_fromversion(self):
+        # type: () -> bool
+        """Validate that a incident field with type grid is from version > 5.5.0"""
+        if self.current_file.get('type') == 'grid':
+            if LooseVersion(self.current_file.get('fromVersion', '0.0.0')) < LooseVersion('5.5.0'):
+                error_message, error_code = Errors.no_minimal_fromversion_in_file('fromVersion', '5.5.0')
```

Comment:
Please add a new proper error in the errors.py file, e.g: "has a from version of... but the proper from version is 5.5.0"
```suggestion
                error_message, error_code = Errors.incident_field_type_grid_minimal_version(current_file_version, '5.5.0')
```